overview.md

```diff
@@ -26,16 +28,14 @@ Detect for Azure DevOps is architected to integrate seamlessly with build and re
 
 ## Documentation ##
 
-Instructions and examples are available on our [Public Confluence](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/622618/Synopsys+Detect+for+Azure+DevOps)
-
-For information on the full capabilities of Detect visit [Detect Public Confluence](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/62423113/Synopsys+Detect)
```

Comment:
Why was this removed?

Reply:
@devmehtabd I updated the URL for the ADO instructions and left out the other one since it is the same document.